my_view_page.php

```diff
@@ -89,13 +89,17 @@
 #print_r ($t_boxes);
 
 $t_project_id = helper_get_current_project();
+
+# Add class "col-md-7" only if the timeline is shown.
 ?>
-<div class="col-md-7 col-xs-12">
+<div class="col-xs-12 <?php echo ( access_has_project_level(config_get( 'timeline_view_threshold' )) ) ? "col-md-7" : "col-md-6"; ?>">
```

Comment:
It would be better to calculate the the class in line 92, then just use a variable here.  Also review the spacing around ().  For example: 

    access_has_project_level( config_get( 'timeline_view_threshold' ) ) )